docs(readme): beautify a bit (#96)
metallang/metal: README.md

# Metal

> A beautiful, robust, flexible, efficient general-purpose programming language with zero garbage collection built with LLVM and Rust.

Reviewer: now that's just false, we don't have any of that

Author: Uh, idk what you mean?

beautiful: it doesn't exist, so the sky is beautiful
robust: you do anything with it, since it doesn't exist
flexible: use it anywhere! it doesn't exist, so it doesn't matter
efficient: it's the fastest programming language ever since, well, it doesn't exist
zero garbage collection: it doesn't exist, so it can't have garbage collection anyway

Reviewer: i'd rather just revert the change

# Metal

> A prototype compiled programming language, striving to be an easy-to-use borrow checked, non-garbage-collected,

Reviewer: ```suggestion
A prototype compiled programming language, striving to be an easy-to-use, borrow checked, non-garbage-collected,
```

## Development

### Rust

The Metal toolchain uses the nightly Rust channel, so to work on it you should use a reasonably recent nightly Rust installation.

> targets supporting it specifically.

Reviewer: ```suggestion
```
oops

## Development

### Rust

> The Metal Compiler, analyzer, and surrounding tools are all built with Rust.
> More specifically, you should use the most latest Rust nightly release as Metal only

Reviewer: ```suggestion
The Metal toolchain uses the nightly Rust channel, so to work on it you should use a reasonably recent nightly Rust installation.
```
